modules/kubernetes-addons/kube-prometheus-stack/outputs.tf

```diff
@@ -0,0 +1,4 @@
+# output "argocd_gitops_config" {
```

Comment:
The commented out bits on gitops - is that because support hasn't been added to https://github.com/aws-samples/eks-blueprints-add-ons ?

Reply:
yes, I am still trying to understand that integration and left the comment in case that was required. removed for now.